.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 *.iml
+*.swp
```

Comment:
It's fine with me, just makes it easier for people using vim not to include (swap) files that they are working on accidentally. I've seen other editor files in the .gitignore, so I felt it would help. But if it's the standard in the hapijs repositories not to include it, then that's absolutely fine with me. But that may mean that you guys need (more) vim users. ;)